src/util/register-service-worker.js

```diff
@@ -1,5 +1,5 @@
 const serviceWorkerUrl =
-  __BUILD__ === "latest" ? "/service_worker.js" : "/service_worker_es5.js";
```

Comment:
These should be reverted. Only ones that are used in either `import` or `import(…)`, as those are resolved by webpack. Others are resolved by the browser and that won't work.